build.sbt

```diff
@@ -15,7 +15,11 @@ sonatypeProjectHosting := Some(GitHubHosting("cb372", "scala-typed-holes", "chri
 scalacOptions ++= Seq("-deprecation")
 scalacOptions in Test ++= {
   val jar = (packageBin in Compile).value
-  Seq(s"-Xplugin:${jar.getAbsolutePath}", s"-Jdummy=${jar.lastModified}") // ensures recompile
+  Seq(
+    s"-Xplugin:${jar.getAbsolutePath}",
+    s"-Jdummy=${jar.lastModified}", // ensures recompile
+    "-P:typed-holes:log-level:info"
```

Comment:
I was wondering if this (`info`) was used as the default, but then I found it [here](https://github.com/cb372/scala-typed-holes/pull/7/files#diff-8ee56c6d8931ba17ae7915dec5f2d6e5R19). Does this mean you've been using `info` yourself or is this just left over from testing? In the former case if both you and I prefer info I wonder if info would be a better default?

Reply:
I left it as info here because the tests, i.e. the example file with lots of holes in it, get compiled as part of the release process for the plugin. Having a whole page of compiler warnings whizz by was freaking me out.

As for what the default should be, I still lean towards warning, but let's discuss it tomorrow 😄